conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+def pytest_addoption(parser):
+    """Add `--runslow` option to py.test."""
+    parser.addoption("--runslow", action="store_true",
```

Comment:
@brittainhard: note that the `--runslow` option must be supplied to run functional tests. Otherwise, a bare `py.test` will skip those tests marked with `@pytest.mark.slow`.